<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionSettingsDialog</class>
 <widget class="QDialog" name="ConnectionSettingsDialog">
  <property name="windowTitle">
   <string>Параметры соединения</string>
  </property>
  <layout class="QFormLayout" name="form_layout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_host">
     <property name="text">
      <string>IP-адрес ПЛК:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="ed_host">
     <property name="placeholderText">
      <string>например, 192.168.0.1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_port">
     <property name="text">
      <string>Порт:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="ed_port">
     <property name="placeholderText">
      <string>например, 502</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_timeout">
     <property name="text">
      <string>Таймаут (с):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="ed_timeout"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_poll">
     <property name="text">
      <string>Период опроса (мс):</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="ed_poll"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>ConnectionSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>ConnectionSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
